config/charmap.json

```diff
@@ -1,7 +1,9 @@
 {
+  "#": "sharp",
```

Comment:
That's not a sharp sign. It renders similarly, but a musical sharp sign is unicode 266F: ♯

Reply:
(What you've provided here is ASCII 35, which is a number sign.)

Reply:
I add this because i am using `next export` in nextjs and with nested slug that contain `#` sign will not working on next export. In my case, slug contain `#` sign then browser will understand that is a URI fragment

Reply:
> I add this because i am using `next export` in nextjs and with nested slug that contain `#` sign will not working on next export. In my case, slug contain `#` sign then browser will understand that it is a URI fragment

I don't understand. `slug` drops the `#` character by default. Are you using the most recent version? Can you give an example of input that is resulting in problematic output for your use case?

Reply:
Sorry, bug only happen on my case. I can use `extend` in `slugify` . I will close this pull request